File: ApexBooking.Core.Domain/Enums/CancellationPolicy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ApexBooking.Core.Domain.Enums
{
    public enum CancellationPolicy
    {
        NoRefund,
        PartialRefund,
        FullRefund
    }
}

File: client-app/src/types/CancellationPolicy.ts
export const CancellationPolicy = {
  NoRefund: 'NoRefund',
  PartialRefund: 'PartialRefund',
  FullRefund: 'FullRefund',
} as const

export type CancellationPolicy = (typeof CancellationPolicy)[keyof typeof CancellationPolicy]
